.evergreen/config.yml

```diff
@@ -2726,8 +2736,15 @@ buildvariants:
 #    coverage: "*"
     encryption: "*"
   display_name: "${encryption} ${python-version} ${platform} ${auth-ssl}"
-  tasks: *encryption-server-versions
-
+  tasks: "test-latest-replica_set"
+  rules:
+    - if:
+        encryption: ["encryption", "encryption_crypt_shared"]
+        platform: ubuntu-18.04
+        auth-ssl: noauth-nossl
+        python-version: "*"
+      then:
+        add_tasks: *encryption-server-versions
```

Comment:
Let's do the same for test-macos-encryption and tests-windows-encryption

Reply:
Done